config/config.go

```diff
@@ -24,7 +25,10 @@ func LoadFile(filename string) (*Config, error) {
 
 var (
 	DefaultModule = Module{
-		Version: 2,
+		Version:        2,
+		Retries:        3,
+		MaxRepititions: 25,
+		Timeout:        60,
```

Comment:
Ah, thanks! The piece I was missing was that yaml.v2 provides a more convenient way of interpreting `time.Duration` using a string value as opposed to just an integer value. Sorry for the misunderstanding. I had not looked very deeply at how yaml.v2 worked. I'll have these adjustments committed shortly.